wombat-admin/src/components/daily_visits.tsx

```diff
@@ -1,23 +1,25 @@
-type RequestProps = {
+type DailyVisitProps = {
   toggleViewImageModal: () => void;
+  users: any
+
 };
-export default function DailyVisits({ toggleViewImageModal }: RequestProps) {
-  const users = [
-    {
-      name: "Sabina Rasulova",
-      role: "Front-end Developer",
-      checkIn: "3/6/2024 5:19 PM",
-      status: "Present",
-      checkOut: "",
-    },
-    {
-      name: "Kenneth Blanton",
-      role: "Full Stack Developer",
-      checkIn: "3/6/2024 5:19 PM",
-      status: "Absent",
-      checkOut: "3/6/2024 6:30 PM",
-    },
-  ];
+export default function DailyVisits({ toggleViewImageModal, users }: DailyVisitProps) {
+  // const users = [
```

Comment:
Nit: Please remove commented out blocks. We have the git history if we ever need to get this back.

Reply:
fixed